### PR TITLE
Optional formatter

### PR DESCRIPTION
**Pull request by zrthxn**
Currently printing `null` for no value
```
(folly::Optional<int>) someInt = null
```

And printing this way for values
```
(folly::Optional<int>) $1 = (int) value = 34

(folly::Optional<folly::basic_fbstring<char, std::char_traits<char>, std::allocator<>, folly::fbstring_core<char> > >) $4 = (folly::basic_fbstring<char, std::char_traits<char>, std::allocator<>, folly::fbstring_core<char> >) value = (store_ = (folly::fbstring) "Hello, World!")
```

### CODE REVIEW

File: src/optional/optional_formatter.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+			.GetChildMemberWithName('storage_')
+			.GetChildMemberWithName('hasValue')
+			.GetValue()
+		) == 'true' else False
+		
+		if has_value:
+			return (valobj
+				.GetChildMemberWithName('storage_')
+				.GetChildMemberWithName('value')
+				# .GetValue()
```
Comment (a20012251) [inline src/optional/optional_formatter.py:18]: remove this

File: src/optional/optional_formatter.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			)
+		else:
+			return "null"
```
Comment (a20012251) [inline src/optional/optional_formatter.py:21]: instead of null print "No value"
this will avoid confusions with the actual null pointer

**Review by walter-erquinigo — CHANGES_REQUESTED** [review]
minor changes needed, but good job!

File: src/optional/optional_formatter.py (hunk 2 above)
Comment (zrthxn) [reply src/optional/optional_formatter.py:21]: Done

**Pull request merged**